JShellAPI/src/test/java/org/togetherjava/jshellapi/JShellApiTests.java

```diff
@@ -0,0 +1,74 @@
+package org.togetherjava.jshellapi;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import org.togetherjava.jshellapi.dto.JShellResult;
+import org.togetherjava.jshellapi.dto.JShellSnippetResult;
+import org.togetherjava.jshellapi.dto.SnippetStatus;
+import org.togetherjava.jshellapi.dto.SnippetType;
+import org.togetherjava.jshellapi.rest.ApiEndpoints;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Integrates tests for JShellAPI.
+ */
+@ContextConfiguration(classes = Main.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class JShellApiTests {
+
+    @Autowired
+    private WebTestClient webTestClient;
+
+    @Autowired
+    private Config appConfig;
```

Comment:
How do you handle custom config ? Like if for one test, you need to use custom config ?

Reply:
I thought about that, but I don't think that would be a good idea as it's an integration-test, and it must verify what exactly happens in real scenario (with the real config).

Reply:
The problem is that the real world config has timers from the minute to the dozen of minutes, you do not want a test to last for 30 minute because you wanted to check if a session correctly disapear after 30 minutes...

Reply:
in that case we may create a common config and separate config